test/functional/ui/float_spec.lua

```diff
@@ -3631,9 +3631,9 @@ describe('float window', function()
         ]], float_pos=expected_pos}
       else
         screen:expect([[
-                    {1:very           }               |
-          {0:~         }{1:^float          }{0:               }|
-                                                  |
+                    {1:such           }               |
+          {0:~         }{1:very           }{0:               }|
+                    ^                              |
```

Comment:
This change defeats the purpose of this test. I think it's better to change this test to run with cmdheight = 0.